audiostream/src/ni/media/audio/sink/wav_sink.h

```diff
@@ -39,7 +39,7 @@ namespace detail
 {
 
 template <class Sink>
-auto writeWavHeader( Sink& sink )
+auto write_wav_header( Sink& sink )
```

Comment:
these method names are better, but maybe instead of `detail`, they should be in a `wav` namespace

Reply:
I don't know if I agree with renaming the namespaces from `detail` to `aiff`/`wav` as they are private implementation details. Renaming the namespace to those names makes it sound like it should be a part of the interface.

Reply:
I agree with Nate

Reply:
sure, maybe like `wav::detail::....` We already see it across projects where you get collisions in the `detail` namespace because every library already has something like `detail::write()` imo it's a silly pattern that's a side effect of C++'s nature

Reply:
i agree, though this header file is not exposed through the public api and even if it was the name `write_wav_header` is not likely to clash with anything else. personally i try to avoid adding too many namespaces whenever possible